src/web/mod.rs

```diff
@@ -50,21 +58,23 @@ pub fn run(soup: Blender) -> HttpResult<Listening> {
     };
 
     for (path, ep) in ins.into_iter() {
-        use std::sync::Mutex;
-        let put = Mutex::new(ep.f);
+        // use std::sync::Mutex;
```

Comment:
because on line 117, we need to Box a Mutex with the Blender to keep it in the global accessible data for the server. without that the blender goes out of scope and channels get closed.